usaspending_api/api_docs/api_documentation/federal_account/spending_by_category.md

```diff
@@ -0,0 +1,58 @@
+## Federal Account Spending By Category - Deprecated
```

Comment:
This endpoint is not used, remove from docs and API

Reply:
Done